### ISSUE TITLE
YePpHa NEW LAYOUT FIX RESPONSE

### ISSUE THREAD

**Issue by omperus** [YePpHa/YouTubeCenter#2416]
I sent a message via Reddit to YePpHa and he responded [this](https://i.imgur.com/ESzqpuP.png)

> from YePpHa sent 3 days ago
>I'm not going to fix YouTube Center for the new YouTube layout.

So thats it then, this script is officialy dead unless someone forks it, a real shame, YouTube seems unusable without it.

**Comment by falcon4fun** [comment]
Sad

**Comment by yllekz** [comment]
That's really disappointing and doesn't feel like an acceptable response. This needs to be fixed.

**Comment by jonnyrobbie** [comment]
Well, it is a MIT license. You have several options. You can either develop a patch and submit it to YePpHa for a pull request. He made quite a few pull requests. Chances are if it will work, you'll get pulled. You can also fork the repo, develop a patch and then hope that people will notice you and start using your patched version. You risk community fragmentation though.

The point is, this is a free software, and you are free to fix it.

**Comment by p1nkyy** [comment]
>  yllekz commented 38 minutes ago
> That's really disappointing and doesn't feel like an acceptable response. This needs to be fixed.

He literally has no obligations to. If he doesn't want to develop this script anymore, it's his prerogative.
But it's sad indeed.

**Comment by andyb2** [comment]
I did the console.log patch on my files and  youtube vids still didnt load. Checkin through tampermonkey shows only console.log

**Comment by ddd111ddd** [comment]
https://github.com/ParticleCore/Particle/issues/493

**Comment by omperus** [comment]
@ddd111ddd Particle (aka Youtube+) doesn't have even half the features YouTubeCenter has  though ☹

**Comment by ddd111ddd** [comment]
@omperus yeah but since it's YePpHa is leaving the project and nobody is going to refactor YouTubeCenter to meet the new material layout requirements, the easiest route is to support Particle. IIRC it was mentioned that the changes are so big you're better of starting from scratch.

I know that nobody is going to fork YT Center this late in the game because this layout change has been going on since may/2016 and nobody stepped up.

I said it before, I'm really grateful for how much this project helped me but it's time to move on. Unless somebody has a better idea or will start a new project based on YTCenter features. 

PS: if you're creating a new project, don't use ANY reference to YouTube in the name. https://github.com/ParticleCore/Particle/issues/483 Some people are saying it was also because of the blacklist an adblocking features, who knows.

**Comment by ddd111ddd** [comment]
lol https://github.com/YePpHa/maia-yt

**Comment by Lunarhawk** [comment]
^ is he basically remaking the script from scratch but with a new name? I'm kind of confused what that is.

**Comment by CollinChaffin** [comment]
@andyb2 did you follow my exact directions in #2410?  If you do and disable the new MD (still an option under YT settings) I can confirm my fix method is still 100% working for me in Chrome and TM if performed correctly without doing anything else.

**Comment by andyb2** [comment]
Yea I followed the directions. I got it working late yesterday after some trial and error. An adblocker I use had removed my whitelist that had youtube in it and was causing problems with youtube. Your fix works fine thanks.

**Comment by jacobs0n** [comment]
how do you revert the new youtube style, then?

**Comment by YePpHa** [comment]
@Lunarhawk It's a codename. It's not going to be named that on release.

**Comment by YePpHa** [comment]
I could probably elaborate on why I'm not going to fix YouTube Center for the new design. The reason is that almost all of the UI features (resize, buttons, like/dislike, comment flags, etc) will need to be redesigned/refactored.
I made a couple of wrong decisions when I developed the current YouTube Center. The first and foremost pressing issue is that it's all contained inside a single file. This made it a lot harder to maintain YouTube Center as it grew. However, it made it a bit faster to test as you didn't need to concatenate the files. Another issue is the performance of the current YouTube Center. It's probably not noticeable in most cases. However, it's still an issue that needs to be resolved.

In conclusion due to having to remake some of YouTube Center to make it work with the new YouTube design I decide I would rather continue on my "remake YouTube Center completely" project aka. [MaiaYT](https://github.com/YePpHa/maia-yt).

**Comment by Magnic** [comment]
^this will earn you a gigantic statue made of marble and gold from my unworthy self.

**Comment by NiGHTsC** [comment]
Let's hope youtube NEW won't be up this year~~~

**Comment by Parsnip** [comment]
So no hope for bugfixes for youtube center regarding the still current youtube layout? Shame.

**Comment by dirtyid** [comment]
https://github.com/ParticleCore/Iridium/releases/tag/0.1.3b

looks like particle/iridium is dead too.

**Comment by Marucins** [comment]
Is the MaiaYT project alive?

**Comment by YePpHa** [comment]
@Marucins Yes, though I haven't worked on it for a while due to some other stuff.

**Comment by marktedrow1** [comment]
YePpHa,

I have been using the YouTube Centers Developers Build userscript 548 for a long time now both in Google Chrome and in Mozilla Firefox and it has always worked extremely well. I have forced the old You Design back so YouTube Centers Developers Build userscript 548 works well again instead of just getting the white YouTube screen. 

Everywhere I look on the internet for help, everyone talks about getting the YouTube Center, which makes the program as popular as Facebook and Twitter and which also makes your name, YePpHa, a household word as well as being a great developer in line with Bill Gates and Microsoft.
YouTube is worthless to me without YTC. I have tried many other programs but nothing works as well. 
I would even buy your program if updated.

However, even with the old YouTube format, I have dash playback disabled as always so the video would buffer really fast to the end, even while watching it and would even continue to buffer when paused, but now the YouTube videos are acting like dash playback is enabled. They are buffering slightly ahead of the video in dashes. Am I doing something wrong? I have also noticed that the video looks and acts the same whether Player Type: HTML5 or Flash is selected. Does this option not longer work?

I always get nervous with major updates of Isp's like the recent Mozilla Firefox Quantum. It is a disappointing program. Most of the add-ons Mozilla had, that I use alot, no longer work with it so I uninstalled it and reinstalled the previous Firefox version on both my PC and cellphone and disabled the automatic updates.  

I read before that you didn't really work on YTC much now after you got a full time job, but alot of people are great fans of YTC and I would love to see it updated.

Sincerely yours,
marktedrow1

**Comment by andyb2** [comment]
He's making another rather than trying to rewrite youtube center
https://github.com/YePpHa/maia-yt

**Comment by marktedrow1** [comment]
I am looking very forward to it.
